.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: main
+on:
+  - pull_request
+  - push
+jobs:
+  main:
+    name: ${{matrix.node}}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{matrix.node}}
+      - run: npm install
+      - run: npm test
+      - uses: codecov/codecov-action@v3
+    strategy:
+      matrix:
+        node:
+          - lts/hydrogen
```

Comment:
Could we use the same Node-version-minimum as in `remark-breaks`?
That way, we can immediately migrate that project to use this, without forcing a major breaking change

Reply:
No, because I used `node:test`. However, I have changed it to `lts/gallium`.

As a side note: I constantly have to look up these names. It would be much clearer to me to use version numbers.

Reply:
I particularly meant in reply of your OP point “Support was dropped for Node.js < 16”.
I think that’s a bad idea, as it forces a major in `remark-breaks`.
I don’t care much for whether tools/CI use Node 14 or 16 (that is, I’d prefer not to, but `node:test` is great).

---

Using the `lts/*` scheme ensures lts versions are used, which is the goal with this. Numbers don’t really mean anything here. See also https://github.com/nodejs/Release/blob/main/CODENAMES.md

Reply:
I agree that they are not common. So I get that they’re not familiar to you. I quite like the channels though :(